edx_proctoring/api.py

```diff
@@ -1235,7 +1235,7 @@ def _check_eligibility_of_enrollment_mode(credit_state):
     # Also make an exception for the honor students to take the "practice exam" as a proctored exam.
     # For the rest of the enrollment modes, None is returned which shows the exam content
     # to the student rather than the proctoring prompt.
-    return credit_state['enrollment_mode'] == 'verified'
+    return credit_state and credit_state['enrollment_mode'] == 'verified'
```

Comment:
If `credit_state` is `None` it will not check `credit_state['enrollment_mode']` in 
``` python 
my_variable = credit_state and credit_state['enrollment_mode'] == 'verified'
```